### DISCUSSION TITLE
Is darkness too dark?

### DISCUSSION THREAD

**Discussion by AJCM-git** [space-wizards/space-station-14#4277] [Questions]
Normally in SS13 even when there are no light source, you can sorta distinguish your character and your surroundings if you try hard, right now if you are in the total darkness you will not be able to distiguish anything unless you use the outline as a guide. Now, here comes the question: Is this a desired behavior? while its true that in real life you cant see anything in total darkness, i dont thing that  adding that as a gimmick is going to do any good
\>But muh  tension and scary
It is more tense being in total darkness and be able to walk and see the thing hunting you at the corner of your eye than cant see shit and be dead the next second

My proposal is: Leave the current darkness levels, except for your interaction circle, in there you should be able to ar least distinguish the things surrounding you
\>wyci

**Comment by ghost** [comment]
I was under the assumption that we would eventually add a very dim light source on your character in the dark, visible to only you.

**Comment by ghost** [comment]
> I was under the assumption that we would eventually add a very dim light source on your character in the dark, visible to only you.

ew

**Comment by SweptWasTaken** [comment]
Glass has been playing too much Terraria

**Comment by Hugal31** [comment]
Not a light-source, visible by other peoples, but rather a "minimum light value" around your character.

**Comment by Devaluated** [comment]
Anyone know how you would do this? 

like is the engine set up for preventing darkness in a certain area or would you have to bodge it with a light source?

**Comment by Androclast** [comment]
Holy necropost

**Comment by deltanedas** [comment]
necro ops also the hdr pr wouldve been good for this as it would let eyes adjust to darkness

**Comment by Brandon-Huu** [comment]
Darkness is not too dark.
